js/src/format/arrow.ts

```diff
@@ -0,0 +1,61 @@
+import { flatbuffers } from 'flatbuffers';
+
+import * as Schema_ from './Schema';
+import * as Message_ from './Message';
+import * as File_ from './Message';
+
+export namespace fb {
+    export import Schema = Schema_.org.apache.arrow.flatbuf;
+    export import Message = Message_.org.apache.arrow.flatbuf;
+    export import File = File_.org.apache.arrow.flatbuf;
+}
```

Comment:
@TheNeuralBit ah I misunderstood how you did these exports. In order to get closure compiler to work, we have to put the generated flatbuffers code in its own folder. We can move the generated files to `format/fb` and just re-export them from this file like you're doing here. I've got this working now in a branch, want me to PR?

Reply:
Ah, actually I spoke too soon. The way TS compiles namespaces to nested IEFEs in JS confuses Closure Compiler's mangler, so all the `fb.Schema.Foo` references get mangled to different names. I remember figuring this out when I first turned on CC, and that's why we do the ugly `import * as Schema_` and `import Type = Schema_.org.apache.arrow.flatbuf.Type` nonsense everywhere. Ugh.